dna/codon.go

```diff
@@ -365,6 +365,19 @@ func TranslateSeqToTer(b []Base) []AminoAcid {
 	return answer
 }
 
+//TranslateToTer takes a sequence of in-frame DNA bases and translates to the first stop codon. Returns a slice of amino acids the distance to the stop codon in base zero.
```

Comment:
This comment talks about a distance to the stop codon, but I don't see that being calculated.